src/lib/validations/sql.ts

```diff
@@ -86,6 +88,30 @@ interface ValidationResult {
 	infos: string[];
 }
 
+/**
+ * Check if a file has a valid extension
+ *
+ * @param {string} filename The file extension
+ * @returns {boolean} True if the extension is valid
+ */
+export const validateImportFileExtension = ( filename: string ): void => {
+	const ext = path.extname( filename ).toLowerCase();
+	if ( ! [ '.sql', '.gz' ].includes( ext ) ) {
```

Comment:
In the public [docs](https://docs.wpvip.com/how-tos/migrate-content-databases/import-with-vip-cli/) we tell folks that we do not support compressed file imports

> A SQL database file (e.g. downloaded from [Database Backups](https://docs.wpvip.com/technical-references/vip-dashboard/backups/)) must already be located on the local machine under the home directory of the current user. The SQL database file must be decompressed (unarchived) before import.

If we are starting to support it, I think we'll need to document the extensions that we support henceforth ?

Reply:
Also any particular reason for not supporting other formats?

Reply:
> If we are starting to support it, I think we'll need to document the extensions that we support henceforth ?

We have always supported `.gz` files in `vip import sql`. We're just aligning `vip dev-env import sql` to support them as well. Agree this should be mentioned in our docs following the release of this. cc @yolih 

The archived must be extracted for `vip import validate-sql`

> Also any particular reason for not supporting other formats?

I think it would be good to support `.zip` as well, but I feel like that's outside the scope of this PR.

Reply:
> The archive must be extracted for vip import validate-sql

better to make this clear in the docs too!